tests/blend.py

```diff
@@ -2,24 +2,34 @@
 
 import pytest
 import numpy as np
-from minerva_lib.blend import to_f32
-from minerva_lib.blend import f32_to_bgr
-from minerva_lib.blend import linear_bgr
+from minerva_lib.blend import make_rgb
```

Comment:
Import multiple functions in one line:
```python
from minerva_lib.blend import make_rgb, linear_rgb
```

Reply:
done `refactor composite_image for single channel`